src/pyhf/infer/utils.py

```diff
@@ -0,0 +1,140 @@
+"""Inference for Statistical Models."""
+
+from .. import get_backend
+from .calculators import AsymptoticCalculator, ToyCalculator
+
+
+def create_calculator(calctype, *args, **kwargs):
+    """
+    Creates a calculator object of the specified `calctype`.
+
+    See ~pyhf.infer.calculators.AsymptoticCalculator and ~pyhf.infer.calculators.ToyCalculator on additional arguments to be specified.
```

Comment:
These also aren't getting rendered out correctly when `utils.create_calculator` is added to `api.rst`.